src/Reducers/AuthReducer.jsx

```diff
@@ -0,0 +1,11 @@
+export function Reducer(state, action) {
+  switch (action.type) {
+    case "SIGNUP":
```

Comment:
You can export these actions as constants as a good practice it will decrease number of typos.

Reply:
I will Update.